airship/AIR-138_kubeadm-rotating-ca.md

```diff
@@ -55,6 +90,13 @@ etcd-server
 front-proxy-client      
 scheduler.conf
 ```
+The same set of observations apply here as stated in [Automatic Certificate Renewal](###Automatic%20Certificate%20Renewal) 
+
+## Key observations
+1. Only client certificate is renewed. 
+2. Even if the CA certificate is soon expiring kubeadm do not renew it automatically. CA rotation has to be done manually.
+3. During the client certificate renewals there is no downtime in service.
+4. The whole process of CA rotation need more experiments to conclude on service downtime.
```

Comment:
We have said twice that there is NO downtime.
What are we looking for in doing further analysis ?

Reply:
Its only the client certificate renewal where there is no downtime. We couldn't conclude since we still don't know the exact way of doing CA rotations since I have seen in the community people have done that through different tweaks